lib/proxy/validations.rb

```diff
@@ -61,6 +63,23 @@ def validate_subnet(subnet)
     subnet
   end
 
+  def validate_cidr(address, prefix)
+    cidr = "#{address}/#{prefix}"
+    network = IPAddr.new(cidr).to_s
+    if network != IPAddr.new(address).to_s
```

Comment:
Is it needed to do a string-based comparison? Can't the `IPAddr` class be compared?
```suggestion
    network = IPAddr.new(cidr)
    if network != IPAddr.new(address)
```

Reply:
Tested this suggestion locally, works as intended